project/src/main/java/com/google/sps/data/User.java

```diff
@@ -183,8 +172,9 @@ public String getEmail() {
     return this.eventsHosting;
   }
 
-  public Set<Long> getEventsHostingIds() {
-    Set<Long> eventIds = new HashSet<>();
+  /** Returns a Set of eventHosting Id fields */
```

Comment:
Actually this comment is not necessary here since the function name explain the purpose of the function.

Comment is needed for servlet functions because the name doesn't explain the function.

Reply:
Done, that makes sense.